Gamma_Isotopes.py

```diff
@@ -7,7 +7,7 @@
 significant gamma emission branching ratios."""
 
 
-class Isotope(object):
+class isotope(object):
```

Comment:
No. Go back and read my last comment on capitalization.
`class Isotope` is correct.